Dart/testData/generate/EqualsAndHashCode20.txt

```diff
@@ -0,0 +1,53 @@
+// Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+class Test {
+  int? a;
+  int? b;
+  int? c;
+  int? d;
+  int? e;
+  int? f;
+  int? g;
+  int? h;
+  int? i;
+  int? j;
+  int? k;
+  int? l;
+  int? m;
+  int? n;
+  int? o;
+  int? p;
+  int? q;
+  int? r;
+  int? s;
+  int? t;
+
+  @override
+  bool operator ==(Object other) =>
```

Comment:
Formatting result is a bit different to me in this and in the ...21.txt file. It's because `dart format` doesn't run in this test after code generation. I updated the expected test data to make the test green. In real life `dart format` does run and the result is different.